internal/characters/baizhu/skill.go

```diff
@@ -101,29 +97,33 @@ func (c *char) chain(src int, count int) combat.AttackCBFunc {
 	}
 }
 
-func (c *char) particleCB(a combat.AttackCB) {
-	if a.Target.Type() != targets.TargettableEnemy {
-		return
-	}
-	if c.StatusIsActive(particleICDKey) {
-		return
-	}
-	count := 3.0
-	if c.Core.Rand.Float64() < 0.50 {
-		count = 4
+func (c *char) makeParticleCB() combat.AttackCBFunc {
+	done := false
+	return func(a combat.AttackCB) {
+		if a.Target.Type() != targets.TargettableEnemy {
+			return
+		}
+		if done {
+			return
+		}
+		done = true
+
+		count := 3.0
+		if c.Core.Rand.Float64() < 0.50 {
+			count = 4
+		}
+		c.Core.QueueParticle(c.Base.Key.String(), count, attributes.Dendro, c.ParticleDelay)
 	}
-	c.AddStatus(particleICDKey, 2*60, true)
-	c.Core.QueueParticle(c.Base.Key.String(), count, attributes.Dendro, c.ParticleDelay)
 }
 
 func (c *char) skillHealing() {
 	c.Core.Tasks.Add(func() {
 		c.Core.Player.Heal(player.HealInfo{
 			Caller:  c.Index,
-			Target:  c.Core.Player.Active(),
+			Target:  -1,
```

Comment:
From skill descriptions: `the Sprite will return, healing all nearby party members based on Baizhu's Max HP.` Sure this should be all and not active only (not clear from dm comparing the logic between other heals)?

Reply:
yes I'm sure. `nearby party members` = everyone